pkg/networkoptions/network_options.go

```diff
@@ -132,11 +132,11 @@ func GetSupportedNetworkOptionsForSubnet(
 	for _, networkOption := range supportedNetworkOptions {
 		isInSidecar := false
 		for networkName := range sc.Networks {
-			if strings.HasPrefix(networkName, networkOption.String()) {
+			if strings.HasPrefix(networkOption.String(), networkName) {
```

Comment:
sorry not clear for me why it was changed. can you pls explain?

Reply:
there was a regression introduced when changing the networkoption string "Fuji" to "Fuji Testnet".
I made this to fix, but was a mistake. Fixing now. Anyway, the network options package implementation
needs to be revised so as can be more simple and easier to understand. but for another PR surely.